testsuite/Main.hs

```diff
@@ -78,6 +78,8 @@ runClashTest =
         , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "LotOfStates"         (["","LotOfStates_testBench"],"LotOfStates_testBench",True)
         -- TODO: Fix for VHDL. ModelSim / GHDL simulate _extremely_ slowly
         , runTest ("tests" </> "shouldwork" </> "Basic") (defBuild \\ [VHDL]) [] "MultipleHidden"      (["","MultipleHidden_testBench"],"MultipleHidden_testBench",True)
+        , runTest ("tests" </> "shouldwork" </> "Basic") defBuild [] "NameInstance"         ([""],"NameInstance_topEntity",False)
+        , outputTest ("tests" </> "shouldwork" </> "Basic") [Verilog]   [] [] "NameInstance" "main"
```

Comment:
Does this only work in Verilog? Or do you trust that if it works in Verilog, it works everywhere?